selfie.c

```diff
@@ -1016,8 +1016,8 @@ void init_memory(uint64_t megabytes) {
 // ------------------------- INSTRUCTIONS --------------------------
 // -----------------------------------------------------------------
 
-void print_source_line_number_of_instruction(uint64_t a);
-void print_instruction_context();
+void print_source_line_number_for_address(uint64_t a);
```

Comment:
Use two procedures `print_code_line_number` and `print_data_line_number`.

Reply:
fixed in 51e3823